adcm_client/objects.py

```diff
@@ -661,6 +664,18 @@ def task_list(self, **args) -> "TaskList":
 
     def run(self, **args) -> "Task":
         with allure_step("Run action {}".format(self.name)):
+            config = self._get_config()
+
+            if 'config' in args:
+                pass
+            elif 'config_diff' in args:
```

Comment:
Please ensure that we have no config and config_diff in the same time.

Reply:
Ok. Only one parameter needed?

Reply:
Yep, only one of them. Because right now we have something strange in line 674 when we have too of them. I don't like line 674 at all